enhancements/automated-service-ca-rotation.md

```diff
@@ -177,6 +183,22 @@ manual rotation (by deleting the signing secret) could be performed. Manual
 rotation would not be seamless, however, due to the lack of a grace period for
 services and clients to converge on the new trust chain.
 
+- Testing in advance of service CA rotation whether consumers of CA artifacts are
+refreshing in a timely manner is complicated:
+ - Unrefreshed clients and servers would be able to communicate without error
+   with both refreshed and unrefreshed clients and servers until the expiry of
+   the pre-rotation CA.
+ - Detecting clients that do not refresh would require starting a cluster with
```

Comment:
I'm less clear how to check for servers that have not refreshed since a refreshed client will have the necessary trust bridge to continue trusting after the expiry of the pre-rotation CA. It may be necessary to fall back on the item starting on line 195.

Reply:
All this reminds me so much of the cert rotation testability in the control plane. You have to wait 24h until installer certs expire. Now way to know earlier whether all parties are updated.